src/shared/components/EditTableForm.tsx

```diff
@@ -283,7 +284,9 @@ const EditTableForm: React.FC<{
     },
     {
       onSuccess: data => {
-        setConfiguration(data);
+        if (isNil(configuration)) {
+          setConfiguration(data);
+        }
```

Comment:
Just curious why this is better than `if(!configuration)`

Reply:
O, just cuz (imho) `isNil` is more deliberate/explicit about what it checks (it checks for `null` and `undefined`) while simple falsy checks can return false for a number of other values.
In any case, this line has now been remoed.